docs/reference/overview_of_strategies.rst

```diff
@@ -1076,58 +956,63 @@ Implementation
 
 HardProber is implemented in the library::
 
-    import axelrod
-    p1 = axelrod.HardProber()  # Create a Prober3 player
-    p2 = axelrod.Cooperator()  # Create a player that always cooperates
-    for round in range(5):
-        p1.play(p2)
-
-    print p2.history
-    print p1.history
-
-which gives::
+    >>> import axelrod
+    >>> p1 = axelrod.HardProber()  # Create a Prober3 player
+    >>> p2 = axelrod.Cooperator()  # Create a player that always cooperates
+    >>> for round in range(5):
+    ...     p1.play(p2)
 
+    >>> p1.history
     ['D', 'D', 'C', 'C', 'D']
+    >>> p2.history
     ['C', 'C', 'C', 'C', 'C']
 
 
 Strategies implemented in the module
 ------------------------------------
 
-There are several original strategies which have been created as part of this project and have never (to our knowledge) appeared in previous tournaments.
+There are several original strategies which have been created as part of this
+project and have never (to our knowledge) appeared in previous tournaments.
 
 Fool Me Once
-^^^^^^^^^^^
+^^^^^^^^^^^^
 
-This strategy begins by cooperating but will defect if at any point the opponent has defected more than once.
+This strategy begins by cooperating but will defect if at any point the opponent
+has defected more than once.
 
 Forgetful Fool Me Once
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Like Fool Me Once, this strategy defects if the opponent ever defects, but sometimes
-forgets that the opponent had defected, cooperating again until another defection.
+Like Fool Me Once, this strategy defects if the opponent ever defects, but
+sometimes forgets that the opponent had defected, cooperating again until
+another defection.
 
 Fool Me Forever
 ^^^^^^^^^^^^^^^
 
-This strategy defects until the opponent defects, and then cooperates there after.
-Note that this strategy is different than opposite grudger, which cooperates
-indefinitely after an opponent cooperation.
+This strategy defects until the opponent defects, and then cooperates there
+after.  Note that this strategy is different than opposite grudger, which
+cooperates indefinitely after an opponent cooperation.
 
 Backstabber
 ^^^^^^^^^^^
 
-Forgives the first 3 defections but on the fourth will defect forever. Defects after the 198th round unconditionally.
+Forgives the first 3 defections but on the fourth will defect forever. Defects
+after the 198th round unconditionally.
 
 DoubleCrosser
 ^^^^^^^^^^^^^
 
-Forgives the first 3 defections but on the fourth will defect forever. If the opponent did not defect in the first 6 rounds the player will cooperate until the 180th round. Defects after the 198th round unconditionally.
+Forgives the first 3 defections but on the fourth will defect forever. If the
+opponent did not defect in the first 6 rounds the player will cooperate until
+the 180th round. Defects after the 198th round unconditionally.
 
```

Comment:
Same comment as for BackStabber - although the 180th round is hard coded, the 198th is not.